subsys/usb/class/mass_storage.c

```diff
@@ -466,7 +467,7 @@ static void CBWDecode(u8_t *buf, u16_t size)
 				} else {
 					usb_ep_set_stall(
 					    CONFIG_MASS_STORAGE_OUT_EP_ADDR);
-					SYS_LOG_DBG("BO-STALL");
+					SYS_LOG_WRN("BO-STALL");
```

Comment:
if this is going to be a warning, we need a proper warning message, what is the warning here?

Reply:
It says that the bulk out endpoint is stalled. Longer logs would affect the behavior even more as it is called here in the ISR context (major issue from the usb stack which needs to be fixed).